packages/modules/web-viewer-react/src/services/Initializer.ts

```diff
@@ -96,12 +95,16 @@ export class WebInitializer {
         const rpcParams: BentleyCloudRpcParams = initializeRpcParams(
           options?.backend
         );
-        // register extensions before startup
-        // TODO only build time ATM
+        yield IModelApp.startup(iModelAppOptions);
+        // register extensions after startup
         if (options?.extensions) {
-          yield addExtensions(options.extensions);
+          options.extensions.map((extension) => {
+            if (extension.hostname) {
+              IModelApp.extensionAdmin.registerHost(`http://${extension.hostname}`);
```

Comment:
do we need http here? could we just use the hostname as is? Most cases would be https anyways

Reply:
registerHost() expects a URL so without the protocol it errors. ... but it would have been better if registerHost() was flexible enough to accept both

Reply:
Ok, I created an[ issue](https://github.com/iTwin/itwinjs-backlog/issues/61) to change that. now that you are parsing the url in the provider constructors, we shouldn't need to do it again. This is ok for now